extensions/hdrhistogram/src/main/java/org/radargun/stats/CommonStatistics.java

```diff
@@ -23,7 +23,7 @@
    private int digits = 2;
 
    public CommonStatistics() {
-      super(new Statistics[] {new BasicStatistics(), new PeriodicStatistics(), new BasicStatistics(new HistogramOperationStats())});
+      super(new Statistics[] {new PeriodicStatistics(), new BasicStatistics(new HistogramOperationStats())});
```

Comment:
You won't get precise values for mean or stddev with histogram stats; buckets introduce imprecision. Ofc it's questionable whether you need more than 2 digits (the default histogram precision)....

Reply:
My motivation behind this commit is that I would like to work with only one data store when looking for statistical values.

Using 2 digits from histogram, 3 digits from histogram and our implementation we have the following values respectively:
5.000021402104251E8
5.000000307106532E8
4.99999999E8

This commit is not mandatory. I would like to follow your advice. 
If you believe that we could have any kind of problem in the future I am fine to revert.